README.md

```diff
@@ -0,0 +1,37 @@
+# An action to start Filebrowser
+
+This action launches an instance of [Filebrowser](https://github.com/filebrowser/filebrowser) and opens it in the default browser.
```

Comment:
This action runs Docker Registry Frontend — https://github.com/neuro-inc/docker-registry-frontend, not Filebrowser